static/js/modals.js

```diff
@@ -264,12 +264,14 @@ function getCustomFields(event) {
   });
 
   const howManyMachinesFieldset = document.getElementById("how-many-machines");
-  const machinesInputs = howManyMachinesFieldset?.querySelectorAll(
-    "input[name='how-many-machines-do-you-have']"
-  );
-  machinesInputs.forEach((input) => {
-    input.removeAttribute("name");
-  });
+  if (howManyMachinesFieldset?.length > 0) {
```

Comment:
Didn't we remove this field before and make it more generic?

Reply:
Yeah good catch, we introduced a new `js-remove-radio-names` on u.com. I'll apply this change to the forms :)